instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/__init__.py

```diff
@@ -153,6 +153,11 @@ def _patched_api_call(self, original_func, instance, args, kwargs):
                             "aws.request_id", req_id,
                         )
 
+                    if "RetryAttempts" in metadata:
+                        span.set_attribute(
+                            "aws.retry_attempts", metadata["RetryAttempts"],
```

Comment:
Is `aws.retry_attempts` the right choice? The [original code](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/150/files#diff-05ef2bbeb79bb2d595bf8a3f7fa2d475d0f34f80804e5e1ba0d9a253d2c0e0e3L154) seems to just have retry_attempts.

That makes sense to me, as retry attempts, albeit behavior in AWS's boto, is not really in aws-specific construct.

retry attempts actually seems quite generic, it probably should be a semantic convention in the spec.